### PR TITLE
Corrected spelling and punctuation errors

### PR DESCRIPTION
**Pull request by s-Hale**
# Fixes # 

### Checklist
- [x] I have made a material change to the repo (functionality, testing, spelling, grammar)
- [x] I have read the [Contribution Guide] and my PR follows them.
- [x] I updated my branch with the master branch.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation about the functionality in the appropriate .md file
- [ ] I have added in line documentation to the code I modified

### Short description of what this PR does:
- Fixes some punctuation errors to improve readability, changed spelling of 'cancelled' to the American English 'canceled' for consistency.
- 

If you have questions, please send an email to [SendGrid](mailto:dx@sendgrid.com), or file a GitHub Issue in this repository.

### CODE REVIEW

**Comment by SendGridDX** [comment]
[CLA assistant check](https://cla.sendgrid.com/sendgrid/sendgrid-python?pullRequest=707) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.sendgrid.com/sendgrid/sendgrid-python?pullRequest=707) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.sendgrid.com/check/sendgrid/sendgrid-python?pullRequest=707) it.</sub>

File: USAGE.md [hunk 1]
```diff
@@ -4572,7 +4572,7 @@ print response.headers
 
 **This endpoint allows you to retrieve the current credit balance for your account.**
 
-Your monthly credit allotment limits the number of emails you may send before incurring overage charges. For more information about credits and billing, please visit our [Classroom](https://sendgrid.com/docs/Classroom/Basics/Billing/billing_info_and_faqs.html).
+Your monthly credit allotment limits the number of emails you may send before incurring coverage charges. For more information about credits and billing, please visit our [Classroom](https://sendgrid.com/docs/Classroom/Basics/Billing/billing_info_and_faqs.html).
```
Comment (misterdorm) [inline USAGE.md]: 'overage' is correct in this context.

Comment (s-Hale) [reply USAGE.md]: Spelling fixed! I didn't know that word, got a litle over-zealous on the corrections. I wasn't 100% sure how to tackle rebasing a PR but I've tried doing a PR against V4.

**Comment by thinkingserious** [comment]
Hello @s-Hale,<br /><br/>Thanks again for the PR!<br />                                              <br />It's <a href="https://sendgrid.com/blog/hacktoberfest-2018-has-arrived/">HACKTOBERFEST</a>! We want to show our appreciation by sending                                              you some special Hacktoberfest swag. If you have not already, could you please fill out                                              <a href="https://dx.sendgrid.com/hacktoberfest/join">this form</a> so we can send it to you? Thanks!                                             <br /><br />Team SendGrid DX

**Comment by s-Hale** [comment]
> [CLA assistant check](https://cla.sendgrid.com/sendgrid/sendgrid-python?pullRequest=707) 
> Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla.sendgrid.com/sendgrid/sendgrid-python?pullRequest=707) before we can accept your contribution.
> **1** out of **2** committers have signed the CLA.
> 
>  s-Hale
>  Sally Hale
> 
> **Sally Hale** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).
> You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.sendgrid.com/check/sendgrid/sendgrid-python?pullRequest=707) it.

I just left a job and no longer have access to that email address... there's a good chance this commit may have been done on that email and I won't be able to verify it.

**Comment by thinkingserious** [comment]
@s-Hale,

Do you mind opening a new PR against the v4 branch with your new GitHub account?

With Best Regards,

Elmer

**Pull request closed without merging**